README

```diff
@@ -185,6 +187,9 @@ Compiler Notes
   for more details:
   https://github.com/open-mpi/ompi/issues/3612
 
+- IBM's xlf compilers: NO known good version that can build/link
+  the MPI f08 bindings or build/link the OpenSHMEM Fortran bindings.
+
```

Comment:
@hppritcha This line is not correct. We regularly test xlf in this way. It is correct the XL and OpenSHMEM have a problem. See note on line 180/182 about that issue. I think this bullet should be removed.